Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -181,65 +164,39 @@ script:
       - certificate
       - browser
       required: true
-      secret: false
-    - default: false
-      description: Value of the exclusion item for the exclusion list.
-      isArray: false
+    - description: Value of the exclusion item for the exclusion list.
       name: exclusion_value
       required: true
-      secret: false
     - auto: PREDEFINED
-      default: false
       description: Operating system type. Required for hash exclusions.
-      isArray: false
       name: os_type
       predefined:
       - windows
       - windows_legacy
       - macos
       - linux
       required: true
-      secret: false
-    - default: false
-      description: Description for adding the exclusion item.
-      isArray: false
+    - description: Description for adding the exclusion item.
       name: description
-      required: false
-      secret: false
     - auto: PREDEFINED
-      default: false
-      description: Exclusion mode (path exclusion only). 
-      isArray: false
+      description: Exclusion mode (path exclusion only).
       name: exclusion_mode
       predefined:
       - suppress
       - disable_in_process_monitor_deep
       - disable_in_process_monitor
       - disable_all_monitors
       - disable_all_monitors_deep
-      required: false
-      secret: false
-    - default: false
-      description: Excluded path for a path exclusion list.
-      isArray: false
+    - description: Excluded path for a path exclusion list.
       name: path_exclusion_type
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of group IDs by which to filter.
+    - description: A comma-separated list of group IDs by which to filter.
       isArray: true
       name: group_ids
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of site IDs by which to filter.
+    - description: A comma-separated list of site IDs by which to filter.
       isArray: true
       name: site_ids
-      required: false
-      secret: false
-    deprecated: false
+      required: true
```

Comment:
Previously while this command did not require group ids and site ids the command would fail if one of the two wasn't provided. In a sense both were mandatory the integration was simply not configured that way. I modified the API call that would be made to work with only site ids as an input while also keeping the behavior the same if both are provided. This shouldn't break anything because if this command was working previously this input would have to be entered.

Reply:
Changed this to no longer be required in the yaml so no longer breaking backwards compatibility